scripts/shared/e2e.sh

```diff
@@ -36,8 +36,8 @@ function test_with_e2e_tests {
 
     ${GO:-go} test -v -timeout 30m -args -test.timeout 15m \
         -submariner-namespace $SUBM_NS "${clusters[@]/#/-dp-context=}" ${gn:+"$gn"} \
-        -ginkgo.v -ginkgo.randomizeAllSpecs -ginkgo.trace \
-        -ginkgo.reportPassed -ginkgo.reportFile "${DAPPER_OUTPUT}/e2e-junit.xml" \
+        --ginkgo.v --ginkgo.randomize-all --ginkgo.trace \
```

Comment:
Do we need any changes in the consuming projects like submariner as well?

Reply:
Yes, and I initially thought everything would be OK because we’d only get this after bumping the Shipyard dependency, but since `e2e.sh` changes all the other Submariner projects will break.

Reply:
I’ve updated the shell scripts to handle both versions of ginkgo and avoid breaking dependent projects.